kilt-modlauncher/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+repositories {
+    mavenCentral()
+    maven("https://maven.neoforged.net/releases")
+    maven("https://maven.fabricmc.net/")
+}
```

Comment:
Aren't these already defined in the buildscript?

Reply:
With #107 this is redundant yes

Reply:
Might be worth removing those then now that #107 has been merged.